.bsp/sbt.json

```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.5.0","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Users/lukejones/Library/Java/JavaVirtualMachines/adopt-openj9-1.8.0_292/Contents/Home/jre/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/lukejones/Library/Application Support/JetBrains/IdeaIC2020.3/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp","--sbt-launch-jar=/Users/lukejones/Library/Application%20Support/JetBrains/IdeaIC2020.3/plugins/Scala/launcher/sbt-launch.jar"]}
```

Comment:
Could you add .bsp to the gitignore please? I had a hard time checking this out because it kept trying to have mine overwrite yours